CMakeLists.txt

```diff
@@ -166,6 +171,7 @@ add_library(HoI4World
 	${HOI4WORLD_SHIP_TYPES_SOURCES}
 	${HOI4WORLD_STATES_SOURCES}
 	${HOI4WORLD_SOUNDS_SOURCES}
+	${HOI4WORLD_TANK_DESIGN_SOURCES}
```

Comment:
Need to also add this to the tests section, but just the new tests files.

Reply:
I have tests files at line 513-517. But there is a typo here (should be _DESIGNS_), maybe that's creating that CQ error?